arm_robots/src/arm_robots/base_robot.py

```diff
@@ -30,6 +31,8 @@ def __init__(self, robot_namespace: str = '', robot_description: str = 'robot_de
         try:
             self.robot_commander = moveit_commander.RobotCommander(ns=self.robot_namespace,
                                                                    robot_description=self.robot_description)
+            self.cartesian = None
```

Comment:
why is this inside the try catch?

Reply:
The controller normally requires joint names/limits from the robot commander, but it's not directly used here so I should just move it outside